frontend/packages/app/src/lib/components/room-settings/room-settings.svelte

```diff
@@ -99,6 +109,14 @@
         on:activate={onAutoRevealChange}
     />
 
+    {#if autoReveal}
+        <label>
+            <span class="time-box-label">Time Box:</span>
+            <input type="number" on:input={onAutoRevealTimerChange} 
```

Comment:
how about we only trigger the function on `change`. I think it only triggers when the user leaves the field and not every time when the values changes. We don't need to know about every value changes